### DISCUSSION TITLE
Vehicle frame

### DISCUSSION THREAD

**Discussion by ravindraji4** [rodralez/NaveGo#45] [Q&A] [answered]
Hello all and Dr. Gonzalez, 

I would like to open the discussion from previous question [https://github.com/rodralez/NaveGo/issues/38](url).

That is correct that my used body-frame IMU should match the convention as in NaveGo. 

1. Is that correct that in NaveGo the vehicle frame and IMU body-frame are in the same direction (**X-forward, Y-right and Z-down**)?

2. Considering NaveGo model. Does it matter if the vehicle moving along X-axis during trajectory 1 and in another case vehicle is moving along Y-axis?

Thank you for your suggestions!

**Comment by rodralez** [comment, marked as answer]
Hello, 

NaveGo can process data coming from 3D trajectories. So, yes, the vehicle can move in any direction.

Best regards.
